src/screens/Error/styles.scss

```diff
@@ -0,0 +1,35 @@
+@import "../../scss/variables";
+@import "../../scss/_base";
+
+#error-screen-div {
```

Comment:
Just make this is a class instead of an #id we'd like to try to avoid using id unless necessary. :)

Reply:
will do!